test/load/allocation/allocationload.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2019 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"os/user"
+	"path/filepath"
+	"sync"
+
+	agonesv1 "agones.dev/agones/pkg/apis/agones/v1"
```

Comment:
nit: can we get rid of the blank lines within vendored dependency library imports please.

Reply:
done